workflow/scripts/gen_new_features.py

```diff
@@ -37,17 +38,20 @@ def main(enhancer_list, abc_predictions, ref_gene_tss, chr_sizes, results_dir):
 
     determine_num_candidate_enh_gene(pred_df, results_dir)
     determine_num_tss_enh_gene(pred_df, ref_gene_tss, results_dir, intermediate_dir)
-    generate_num_sum_enhancers(abc_predictions, enhancer_list, chr_sizes, results_dir, intermediate_dir)
+    generate_num_sum_enhancers(
```

Comment:
for this function/feature, do we want to make the distance range we compute over a parameter? we had previously compared within 5kb and within 10kb and chosen 5kb (i think?) but i believe we still compute both features-- for now we could just compute the 5kb versions since this is a computationally expensive step?

Reply:
That makes sense. That's probably better to do as a separate PR

Reply:
actually i'll just remove the 10kb in this PR since I'm already re-running the pipeline for testing anyways